iot-hub/Samples/device/X509DeviceCertWithChainSample/Parameters.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.Azure.Devices.Client;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace X509DeviceCertWithChainSample
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'h',
+            "hostName",
+            Required = true,
+            HelpText = "The hostname of IotHub.")]
+        public string HostName { get; set; }
+
+        [Option(
+            'd',
+            "deviceName",
+            Required = true,
+            HelpText = "The name of the device.")]
+        public string DeviceName { get; set; }
+
+        [Option(
+            'p',
+            "devicePfxPassword",
+            Required = true,
+            HelpText = "The password of device certificate.")]
+        public string DevicePfxPassword { get; set; }
+
+        [Option(
+            'r',
+            "rootCertPath",
+            Required = true,
+            HelpText = "Path to rootCA certificate.")]
+        public string RootCertPath { get; set; }
+
+        [Option(
+            'i',
```

Comment:
I'd rather not have short names for parameters that have no alphabetic correlation (i, j, k).